src/downloads.jl

```diff
@@ -0,0 +1,56 @@
+"""
+```julia
+download_tiger(output_dir; 
+    year = 2020, 
+    layer = "state"
+)
+```
+Downloads TIGER/Line geographic data from the US Census Bureau for the specified year and geographic layer,
+saving the data as shapefiles.
```

Comment:
not quite - seems it saves as zip files?  Shapefile.jl has zip support via ZipFile.jl but it's a bit dicey.